pkg/ingester/metrics_test.go

```diff
@@ -480,6 +505,29 @@ func TestTSDBMetricsWithRemoval(t *testing.T) {
 			# TYPE cortex_ingester_tsdb_out_of_order_samples_appended_total counter
 			cortex_ingester_tsdb_out_of_order_samples_appended_total{user="user1"} 3
 			cortex_ingester_tsdb_out_of_order_samples_appended_total{user="user2"} 3
+
+			# HELP cortex_ingester_ephemeral_head_gc_duration_seconds Runtime of garbage collection in the TSDB headfor ephemeral storage.
+			# TYPE cortex_ingester_ephemeral_head_gc_duration_seconds summary
+			cortex_ingester_ephemeral_head_gc_duration_seconds_sum 5.154812e+06
+        	cortex_ingester_ephemeral_head_gc_duration_seconds_count 3
+
+			# HELP cortex_ingester_ephemeral_head_truncations_failed_total Total number of TSDB head truncations that failedfor ephemeral storage.
```

Comment:
there are some spaces missing now before `for`

Reply:
Thanks! :)